### ISSUE TITLE
Label setAlignment(Align.center) doesn't work properly with setEllipsis(true)

### ISSUE THREAD

**Issue by geertjan-apptornado** [libgdx/libgdx#3045]
When on a Label you set both `setAlignment(Align.center)` and `setEllipsis(true)`, the alignment is broken when the text doesn't fit.

#### Expected behaviour

When text fits:
```
    +--------+
    |  1234  |
    +--------+
```

When text doesn't fit:
```
    +--------+
    |12345...|
    +--------+
```

#### Actual behavior (libGDX 1.5.6)

When text fits:
```
    +--------+
    |  1234  |
    +--------+
```

When text doesn't fit:
```
    +--------+
 12345...    |
    +--------+
```

If you use `setAlignment(Align.right)` you get similar incorrect behavior.

**Comment by NathanSweet** [comment]
Fixed in 033a9469a4fce472468da3b0062303dc1248a2be

**Comment by NathanSweet** [comment]
Sorry, I lied. Gotta run atm, will fix later.
